index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
> TYPE_ROTATION_VECTOR should suffice for all use-cases covered by TYPE_GEOMAGNETIC_ROTATION_VECTOR and TYPE_GAME_ROTATION_VECTOR

I'm pretty sure that's incorrect, but I'd like to hear the opinion of people more knowledgeable about this topic than I, e.g. @borismus, who [wrote the following](http://smus.com/sensor-fusion-prediction-webvr/) (great article btw) about the original device orientation event:

> "In Android, the JavaScript deviceorientation event was implemented using Sensor.TYPE_ORIENTATION in Android, which fuses accelerometer, gyroscope and magnetometer sensors together to give a North-aligned orientation. The trouble is that the magnetometer's estimate of magnetic North is easily affected by external metallic objects. On many devices, the North estimate continually changes, even when you are not looking around. This breaks the correspondence between motion and display, a recipe for disaster."

Now I'm aware of the distinction between TYPE_ORIENTATION and TYPE_ROTATION_VECTOR (it's essentially a *data format* distinction), but the issue here is about the physical characteristics of motion sensors. I don't see how the characteristics of magnetometers would have changed in two years making the above no longer relevant (nor would that help us deal with deployed devices had they done so anyway).